tests/test_lambda.py

```diff
@@ -0,0 +1,307 @@
+import hashlib
```

Comment:
Integrating the test into the main `test_smoke.py` would be nice before we merge this PR.

Reply:
My worry is that lambda tests cannot run concurrently (API rate limits), but we usually run multiple smoke tests at once.